No path for wallet has been provided

<!--- Provide a general summary of the issue in the Title above -->

## Description
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug. What were you hoping to achieve? -->
Installing v1.0.13/14 from an older version threw the following error:

```
10/28/2019 11:19:26 AM] [INFO] Tidying wallet, gateway, environment and repository settings
[10/28/2019 11:19:26 AM] [IMPORTANT] Log files can be found by running the `Developer: Open Logs Folder` command from the palette

[10/28/2019 11:19:26 AM] [INFO] Starting IBM Blockchain Platform Extension
[10/28/2019 11:19:28 AM] [INFO] Attempting to require dependency: grpc
[10/28/2019 11:19:28 AM] [INFO] Error requiring dependency: Failed to load gRPC binary module because it was not installed for the current system
Expected directory: node-v69-darwin-x64-unknown
Found: [node-v64-linux-x64-glibc]
This problem can often be fixed by running "npm rebuild" on the current system
Original error: Cannot find module '/Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/node_modules/grpc/src/node/extension_binary/node-v69-darwin-x64-unknown/grpc_node.node'
[10/28/2019 11:19:29 AM] [INFO] Updating native node modules
[10/28/2019 11:19:29 AM] [INFO] Rebuilding native node modules
[10/28/2019 11:19:31 AM] [INFO] > grpc@1.23.3 install /Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/node_modules/grpc
[10/28/2019 11:19:31 AM] [INFO] > node-pre-gyp install --fallback-to-build --library=static_library
[10/28/2019 11:19:31 AM] [INFO] node-pre-gyp
[10/28/2019 11:19:31 AM] [INFO]  WARN Using request for node-pre-gyp https download
[10/28/2019 11:19:31 AM] [INFO]
[10/28/2019 11:19:31 AM] [INFO] [grpc] Success: "/Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/node_modules/grpc/src/node/extension_binary/electron-v4.2-darwin-x64-unknown/grpc_node.node" is installed via remote
[10/28/2019 11:19:31 AM] [INFO] grpc@1.23.3 /Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/node_modules/grpc
[10/28/2019 11:19:31 AM] [INFO] Updating grpc
[10/28/2019 11:19:31 AM] [SUCCESS] Finished updating native node modules
[10/28/2019 11:19:31 AM] [INFO] Rewriting activation events
[10/28/2019 11:19:31 AM] [INFO] Clearing extension cache
[10/28/2019 11:19:31 AM] [INFO] Migrating local runtime manager
[10/28/2019 11:19:31 AM] [INFO] Initializing local runtime manager
[10/28/2019 11:19:32 AM] [ERROR] Failed to activate extension: Error: No path for wallet has been provided
[10/28/2019 11:19:32 AM] [ERROR] Error: No path for wallet has been provided
    at new FileSystemWallet (/Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/node_modules/fabric-network/lib/impl/wallet/filesystemwallet.js:65:10)
    at new FabricWallet (/Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/build/extension/fabric/FabricWallet.js:30:9)
    at FabricWalletGenerator.<anonymous> (/Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/build/extension/fabric/FabricWalletGenerator.js:34:20)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/bmosus.ibm.com/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.13/build/extension/fabric/FabricWalletGenerator.js:18:58)
```

I suspect this is to do with migration of settings and registries. 

[extDir.zip](https://github.com/IBM-Blockchain/blockchain-vscode-extension/files/3783587/extDir.zip)

User settings:
```
{
    "fabric.runtimes": [
        {
            "name": "local_fabric",
            "developmentMode": false,
            "ports": {
                "orderer": 17050,
                "peerRequest": 17051,
                "peerChaincode": 17052,
                "peerEventHub": 17053,
                "certificateAuthority": 17054,
                "couchDB": 17055
            }
        }
    ],
    "blockchain.repositories": [
        {
            "name": "hyperledger/fabric-samples",
            "path": "/Users/bmosus.ibm.com/Documents/fabric-samples"
        }
    ],
    "fabric.gateways": [
        {
            "name": "BarryCluster",
            "connectionProfilePath": "/Users/bmosus.ibm.com/.fabric-vscode/BarryCluster/connection.json",
            "associatedWallet": "BarryWallet"
        }
    ],
    "fabric.runtime": {
        "ports": {
            "orderer": 17050,
            "peerRequest": 17051,
            "peerChaincode": 17052,
            "peerEventHub": 17053,
            "certificateAuthority": 17054,
            "couchDB": 17055,
            "logs": 17056
        },
        "developmentMode": true
    },
    "fabric.wallets": [
        {
            "managedWallet": false,
            "name": "BarryWallet",
            "walletPath": "/Users/bmosus.ibm.com/.fabric-vscode/BarryWallet"
        }
    ],
    "ibm-blockchain-platform.fabric.gateways": [],
    "ibm-blockchain-platform.fabric.wallets": [],
    "ibm-blockchain-platform.ext.repositories": [],
    "ibm-blockchain-platform.fabric.runtime": {
        "ports": {
            "orderer": 17050,
            "peerRequest": 17051,
            "peerChaincode": 17052,
            "peerEventHub": 17053,
            "certificateAuthority": 17054,
            "couchDB": 17055,
            "logs": 17056
        },
        "developmentMode": true
    },
    "window.zoomLevel": 1,
    "diffEditor.ignoreTrimWhitespace": true,
    "ibm-blockchain-platform.fabric.environments": [],
    "git.confirmSync": false,
    "git.autofetch": true,
    "ibm-blockchain-platform.ext.bypassPreReqs": true
}
```
## Expected Behavior
<!--- Tell us what should happen -->

## Actual Behavior
<!--- Tell us what happens instead -->

## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- Reproduce this bug include code to reproduce, if relevant -->
1.
2.
3.

## Existing issues
<!-- Have you searched for any existing issues or are their any similar issues that you've found? -->
- [ ] [Stack Overflow issues](https://stackoverflow.com/questions/tagged/ibp-vscode-extension)
- [ ] [GitHub Issues](https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues)

<!-- Please include any links to issues here -->

## Log File

<!-- In Visual Studio Code's command palette, run '>Developer: Open Logs Folder' -->
<!-- Attach/link to the latest nested 'Blockchain.log' file -->

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* IBM Blockchain Platform Extension Version:
* Visual Studio Code version:
* Environment name and version (e.g. Node.js v8.12.0, npm v6.4.1):
* Operating System and version:
* Link to your project:

> ### Failed to activate extension: No path for wallet has been provided
> I can't start IBM blockchain platform vscode extension.
> 
> > [10/29/2019 6:56:05 PM] [IMPORTANT] Log files can be found by running the `Developer: Open Logs Folder` command from the palette
> 
> > [10/29/2019 6:56:05 PM] [INFO] Starting IBM Blockchain Platform Extension
> 
> > [10/29/2019 6:56:10 PM] [INFO] Attempting to require dependency: grpc
> 
> > [10/29/2019 6:56:10 PM] [INFO] Migrating local runtime manager
> 
> > [10/29/2019 6:56:10 PM] [INFO] Initializing local runtime manager
> 
> > [10/29/2019 6:56:10 PM] [ERROR] Failed to activate extension: Error: No path for wallet has been provided
> 
> > [10/29/2019 6:56:10 PM] [ERROR] Error: No path for wallet has been provided
> 
> ```
> at new FileSystemWallet (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\node_modules\fabric-network\lib\impl\wallet\filesystemwallet.js:65:10)
> at new FabricWallet (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\build\extension\fabric\FabricWallet.js:30:9)
> at FabricWalletGenerator.<anonymous> (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\build\extension\fabric\FabricWalletGenerator.js:34:20)
> at Generator.next (<anonymous>)
> at fulfilled (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\build\extension\fabric\FabricWalletGenerator.js:18:58)
> ```

Hello @Raumo0 ,
What version of the extension have you upgraded from?
Would it be possible to see your User Settings?

This can be seen by running `>Preferences: Open Settings (JSON)` from the command palette.

> [10/29/2019 3:58:10 PM] [INFO] Starting IBM Blockchain Platform Extension
[10/29/2019 3:58:12 PM] [INFO] Attempting to require dependency: grpc
[10/29/2019 3:58:12 PM] [INFO] Migrating local runtime manager
[10/29/2019 3:58:12 PM] [INFO] Initializing local runtime manager
[10/29/2019 3:58:12 PM] [ERROR] Failed to activate extension: Error: No path for wallet has been provided
[10/29/2019 3:58:12 PM] [ERROR] Error: No path for wallet has been provided
	at new FileSystemWallet (/Users/jschmitz/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.14/node_modules/fabric-network/lib/impl/wallet/filesystemwallet.js:65:10)
	at new FabricWallet (/Users/jschmitz/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.14/build/extension/fabric/FabricWallet.js:30:9)
	at FabricWalletGenerator.<anonymous> (/Users/jschmitz/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.14/build/extension/fabric/FabricWalletGenerator.js:34:20)
	at Generator.next (<anonymous>)
	at fulfilled (/Users/jschmitz/.vscode/extensions/ibmblockchain.ibm-blockchain-platform-1.0.14/build/extension/fabric/FabricWalletGenerator.js:18:58)

> > ### Failed to activate extension: No path for wallet has been provided
> > I can't start IBM blockchain platform vscode extension.
> > > [10/29/2019 6:56:05 PM] [IMPORTANT] Log files can be found by running the `Developer: Open Logs Folder` command from the palette
> > 
> > 
> > > [10/29/2019 6:56:05 PM] [INFO] Starting IBM Blockchain Platform Extension
> > 
> > 
> > > [10/29/2019 6:56:10 PM] [INFO] Attempting to require dependency: grpc
> > 
> > 
> > > [10/29/2019 6:56:10 PM] [INFO] Migrating local runtime manager
> > 
> > 
> > > [10/29/2019 6:56:10 PM] [INFO] Initializing local runtime manager
> > 
> > 
> > > [10/29/2019 6:56:10 PM] [ERROR] Failed to activate extension: Error: No path for wallet has been provided
> > 
> > 
> > > [10/29/2019 6:56:10 PM] [ERROR] Error: No path for wallet has been provided
> > 
> > 
> > ```
> > at new FileSystemWallet (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\node_modules\fabric-network\lib\impl\wallet\filesystemwallet.js:65:10)
> > at new FabricWallet (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\build\extension\fabric\FabricWallet.js:30:9)
> > at FabricWalletGenerator.<anonymous> (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\build\extension\fabric\FabricWalletGenerator.js:34:20)
> > at Generator.next (<anonymous>)
> > at fulfilled (c:\Users\admin\.vscode\extensions\ibmblockchain.ibm-blockchain-platform-1.0.14\build\extension\fabric\FabricWalletGenerator.js:18:58)
> > ```
> 
> Hello @Raumo0 ,
> What version of the extension have you upgraded from?
> Would it be possible to see your User Settings?
> 
> This can be seen by running `>Preferences: Open Settings (JSON)` from the command palette.

Name: IBM Blockchain Platform
Id: ibmblockchain.ibm-blockchain-platform
Description: End to end extension for Hyperledger Fabric developers. Develop and test your blockchain smart contracts and client applications on your local machine, and package your projects for deployment into IBM Blockchain Platform runtimes.
Version: 1.0.14
Publisher: IBM Blockchain
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=IBMBlockchain.ibm-blockchain-platform

@Jakeeyturner I attempted your fix and now I'm seeing this:

```
[10/30/2019 8:37:46 AM] [INFO] Initializing local runtime manager
[10/30/2019 8:37:47 AM] [ERROR] Failed to activate extension: Error: EROFS: read-only file system, mkdir '/~'
[10/30/2019 8:37:47 AM] [ERROR] Error: EROFS: read-only file system, mkdir '/~'
```
Mac OS Catalina 10.15

VSCode Version: 1.39.2
Commit: 6ab598523be7a800d7f3eb4d92d7ab9a66069390
Date: 2019-10-15T15:33:00.827Z
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Darwin x64 19.0.0

IBP Plugin V: 1.0.14

any tips?

--- a/extension/fabric/FabricRuntime.ts
+++ b/extension/fabric/FabricRuntime.ts
@@ -116,6 +116,17 @@ export class FabricRuntime extends FabricEnvironment {
                 walletRegistryEntry.walletPath = path.join(homeExtDir, FileConfigurations.FABRIC_WALLETS, walletName);
                 walletRegistryEntry.managedWallet = true;
                 await FabricWalletRegistry.instance().add(walletRegistryEntry);
+            } else {
+                // Fallback solution if FabricWalletUtil.tidyWalletSettings() fix for "No path for wallet has been provided" doesn't work - https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues/1593
+                // I think the problem occurred because we weren't setting a walletPath or managedWallet in FabricWalletUtil.tidyWalletSettings().
+                const walletRegistryEntry: FabricWalletRegistryEntry = await FabricWalletRegistry.instance().get(walletName);
+                if (!walletRegistryEntry.walletPath || !walletRegistryEntry.managedWallet) {
+                    const extDir: string = vscode.workspace.getConfiguration().get(SettingConfigurations.EXTENSION_DIRECTORY);
+                    const homeExtDir: string = FileSystemUtil.getDirPath(extDir);
+                    walletRegistryEntry.walletPath = path.join(homeExtDir, FileConfigurations.FABRIC_WALLETS, walletName);
+                    walletRegistryEntry.managedWallet = true;
+                    await FabricWalletRegistry.instance().update(walletRegistryEntry);
+                }
             }
 
             const localWallet: IFabricWallet = await fabricWalletGenerator.getWallet(walletName);

--- a/extension/fabric/FabricWalletUtil.ts
+++ b/extension/fabric/FabricWalletUtil.ts
@@ -68,12 +68,15 @@ export class FabricWalletUtil {
         const localWalletExists: boolean = await fs.pathExists(localFabricWalletPath);
         if (localWalletExists) {
             try {
+                newWalletDir = path.join(walletsExtDir, FabricWalletUtil.LOCAL_WALLET);
+
                 // create the new registry entry
                 const walletRegistryEntry: FabricWalletRegistryEntry = new FabricWalletRegistryEntry();
                 walletRegistryEntry.name = FabricWalletUtil.LOCAL_WALLET;
+                walletRegistryEntry.walletPath = newWalletDir;
+                walletRegistryEntry.managedWallet = true;
                 await FabricWalletRegistry.instance().add(walletRegistryEntry);
 
-                newWalletDir = path.join(walletsExtDir, FabricWalletUtil.LOCAL_WALLET);
                 await fs.copy(localFabricWalletPath, newWalletDir);
             } catch (error) {
                 throw new Error(`Issue copying ${localFabricWalletPath} to ${newWalletDir}: ${error.message}`);

--- a/test/fabric/FabricRuntime.test.ts
+++ b/test/fabric/FabricRuntime.test.ts
@@ -184,6 +184,30 @@ describe('FabricRuntime', () => {
             mockFabricWallet.importIdentity.should.have.been.calledOnceWithExactly(sinon.match.string, sinon.match.string, 'admin', 'Org1MSP');
         });
 
+        it(`should include a walletPath and managedRuntime options if not present for ${FabricWalletUtil.LOCAL_WALLET}`, async () => {
+            const mockFabricWalletGenerator: sinon.SinonStubbedInstance<IFabricWalletGenerator> = sandbox.createStubInstance(FabricWalletGenerator);
+            sandbox.stub(FabricWalletGeneratorFactory, 'createFabricWalletGenerator').returns(mockFabricWalletGenerator);
+            const mockFabricWallet: sinon.SinonStubbedInstance<IFabricWallet> = sandbox.createStubInstance(FabricWallet);
+            mockFabricWalletGenerator.getWallet.returns(mockFabricWallet);
+            sandbox.stub(FabricWalletRegistry.instance(), 'exists').withArgs(FabricWalletUtil.LOCAL_WALLET).resolves(true);
+            sandbox.stub(FabricWalletRegistry.instance(), 'get').withArgs(FabricWalletUtil.LOCAL_WALLET).resolves(
+                {
+                    name: FabricWalletUtil.LOCAL_WALLET
+
+                } as FabricWalletRegistryEntry
+            );
+            const updateStub: sinon.SinonStub = sandbox.stub(FabricWalletRegistry.instance(), 'update').resolves();
+            await runtime.importWalletsAndIdentities();
+            updateStub.should.have.been.calledWith({
+                name: FabricWalletUtil.LOCAL_WALLET,
+                walletPath: path.join(TestUtil.EXTENSION_TEST_DIR, 'wallets', FabricWalletUtil.LOCAL_WALLET),
+                managedWallet: true
+            });
+            mockFabricWalletGenerator.getWallet.should.have.been.calledOnceWithExactly(FabricWalletUtil.LOCAL_WALLET);
+            mockFabricWallet.importIdentity.should.have.been.calledOnceWithExactly(sinon.match.string, sinon.match.string, 'admin', 'Org1MSP');
+
+        });
+
     });
 
     describe('#importGateways', () => {

--- a/test/fabric/FabricWalletUtil.test.ts
+++ b/test/fabric/FabricWalletUtil.test.ts
@@ -35,7 +35,7 @@ describe('FabricWalletUtil', () => {
     let fsRemoveStub: sinon.SinonStub;
     let walletA: any;
     let walletB: any;
-
+    let addSpy: sinon.SinonSpy;
     before(async () => {
         await FabricWalletRegistry.instance().clear();
         await TestUtil.setupTests(mySandBox);
@@ -69,6 +69,8 @@ describe('FabricWalletUtil', () => {
             getSettingsStub.withArgs(SettingConfigurations.EXTENSION_DIRECTORY).returns(TestUtil.EXTENSION_TEST_DIR);
             fsCopyStub = mySandBox.stub(fs, 'copy').resolves();
             fsRemoveStub = mySandBox.stub(fs, 'remove').resolves();
+
+            addSpy = mySandBox.spy(FabricWalletRegistry.instance(), 'add');
         });
 
         afterEach(async () => {
@@ -79,9 +81,10 @@ describe('FabricWalletUtil', () => {
             await FabricWalletUtil.tidyWalletSettings();
 
             getSettingsStub.should.have.been.calledWith(SettingConfigurations.OLD_FABRIC_WALLETS);
-
             fsCopyStub.should.not.have.been.called;
-
+            addSpy.should.have.been.calledTwice;
+            addSpy.getCall(0).should.have.been.calledWithExactly(walletA);
+            addSpy.getCall(1).should.have.been.calledWithExactly({name: walletB.name, walletPath: walletB.walletPath});
             await FabricWalletRegistry.instance().getAll().should.eventually.deep.equal([{ name: walletA.name, walletPath: walletA.walletPath }, { name: walletB.name, walletPath: walletB.walletPath }]);
         });
 
@@ -92,20 +95,25 @@ describe('FabricWalletUtil', () => {
             const dest: string = path.join(TestUtil.EXTENSION_TEST_DIR, FileConfigurations.FABRIC_WALLETS, walletA.name);
             fsCopyStub.should.have.been.calledOnceWithExactly(walletA.walletPath, dest);
             fsRemoveStub.should.have.been.calledOnceWithExactly(walletA.walletPath);
+            addSpy.should.have.been.calledTwice;
+            addSpy.getCall(0).should.have.been.calledWithExactly(walletA);
+            addSpy.getCall(1).should.have.been.calledWithExactly({name: walletB.name, walletPath: walletB.walletPath});
 
             await FabricWalletRegistry.instance().getAll().should.eventually.deep.equal([{ name: walletA.name, walletPath: walletA.walletPath }, { name: walletB.name, walletPath: walletB.walletPath }]);
         });
 
-        it('should not create registyr if already exists', async () => {
+        it('should not create registry if already exists', async () => {
             walletA.walletPath = path.join(TestUtil.EXTENSION_TEST_DIR, 'anotherWallet', 'walletA');
 
             await FabricWalletRegistry.instance().add(walletA);
+            addSpy.resetHistory();
+
             await FabricWalletUtil.tidyWalletSettings();
 
             const dest: string = path.join(TestUtil.EXTENSION_TEST_DIR, FileConfigurations.FABRIC_WALLETS, walletA.name);
             fsCopyStub.should.have.been.calledOnceWithExactly(walletA.walletPath, dest);
             fsRemoveStub.should.have.been.calledOnceWithExactly(walletA.walletPath);
-
+            addSpy.getCall(0).should.have.been.calledWithExactly({name: walletB.name, walletPath: walletB.walletPath});
             await FabricWalletRegistry.instance().getAll().should.eventually.deep.equal([{ name: walletA.name, walletPath: walletA.walletPath }, { name: walletB.name, walletPath: walletB.walletPath }]);
         });
 
@@ -125,6 +133,7 @@ describe('FabricWalletUtil', () => {
             await fs.ensureDir(path.join(TestUtil.EXTENSION_TEST_DIR, FabricWalletUtil.LOCAL_WALLET));
             getSettingsStub.withArgs(SettingConfigurations.OLD_FABRIC_WALLETS).returns([]);
             await FabricWalletUtil.tidyWalletSettings();
+            addSpy.getCall(0).should.have.been.calledWithExactly({name: FabricWalletUtil.LOCAL_WALLET, walletPath: path.join(TestUtil.EXTENSION_TEST_DIR, FileConfigurations.FABRIC_WALLETS, FabricWalletUtil.LOCAL_WALLET), managedWallet: true});
 
             fsCopyStub.should.have.been.calledOnceWithExactly(path.join(TestUtil.EXTENSION_TEST_DIR, FabricWalletUtil.LOCAL_WALLET), path.join(TestUtil.EXTENSION_TEST_DIR, FileConfigurations.FABRIC_WALLETS, FabricWalletUtil.LOCAL_WALLET));
             fsRemoveStub.should.have.been.calledOnceWithExactly(path.join(TestUtil.EXTENSION_TEST_DIR, FabricWalletUtil.LOCAL_WALLET));
@@ -136,7 +145,7 @@ describe('FabricWalletUtil', () => {
             const error: Error = new Error('a problem');
             fsCopyStub.rejects(error);
             await FabricWalletUtil.tidyWalletSettings().should.be.rejectedWith(`Issue copying ${path.join(TestUtil.EXTENSION_TEST_DIR, FabricWalletUtil.LOCAL_WALLET)} to ${path.join(TestUtil.EXTENSION_TEST_DIR, FileConfigurations.FABRIC_WALLETS, FabricWalletUtil.LOCAL_WALLET)}: ${error.message}`);
-
+            addSpy.getCall(0).should.have.been.calledWithExactly({name: FabricWalletUtil.LOCAL_WALLET, walletPath: path.join(TestUtil.EXTENSION_TEST_DIR, FileConfigurations.FABRIC_WALLETS, FabricWalletUtil.LOCAL_WALLET), managedWallet: true});
             fsCopyStub.should.have.been.calledOnceWithExactly(path.join(TestUtil.EXTENSION_TEST_DIR, FabricWalletUtil.LOCAL_WALLET), path.join(TestUtil.EXTENSION_TEST_DIR, FileConfigurations.FABRIC_WALLETS, FabricWalletUtil.LOCAL_WALLET));
 
             fsRemoveStub.should.not.have.been.called;

